contents/blog/interview-snapshot-guide.md

```diff
@@ -0,0 +1,69 @@
+---
+date: 2023-02-23
+title: "An 80/20 approach for sharing user interview insights with your team"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - annika-schmid
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: Engineering
+tags:
+  - Guides
+---
+
+Regularly talking to users is an important habit for any team, but you also need an efficient way to share what you learn. Traditional scientific methods areeffective but too time consuming, especially for a fast-moving startup. This is why, at PostHog, we use interview snapshots to share insights from interviews with the team. 
+
+Championed by Teresa Torres in the book [Continuous Discovery Habits](https://www.amazon.co.uk/Continuous-Discovery-Habits-Discover-Products/dp/1736633309), interview snapshots are great conversation starters you can create with minimal effort – an 80/20 method that allows us to gather feedback, generate quick insights, build MVPs, measure their success and iterate. 
+
+In this guide, I'll share how we create user interview snapshots at PostHog – including the following template that you can download and fill out, or recreate in Google Docs, Figma or any other tool of your choice.
+
+// Annika to create this
+
+## Interview snapshots explained
+
+To create a user interview snapshot, you need a list of insights from a user interview. Ideally you take those notes during the interview, or you can ask one of your colleagues to do so. 
+
+While taking notes, focus on how the user is using your product, and how they feel while using it. What makes them excited? What frustrates them? Those are the insights that later go into your interview snapshot.
+
+Another approach is to record the conversation, transcribe and summarise it. This approach is more time consuming, so we don’t recommend it. Though you might want to record the conversation in addition to taking notes, if later on you are looking for word-by-word quotes.
+
+Once you have your insights available, you can open up your interview snapshot template and start filling it out. From experience, we recommend blocking 30 minutes after your interview to review your notes and fill out the user interview snapshot. Here is a step by step guide on how to fill it out:
+
+### 1. Note high-level information of your user
+
+To set the scene, add the user’s name at the top of the interview snapshot. For B2B companies, we also recommend adding the job role and company name. It’s also handy to add the date of the interview and a high-level summary what the interview was about. If available, add a picture of the user and the company logo, if the context is B2B. Believe us, this will spur your memory if you revisit the interview in a couple weeks’ time.
```

Comment:
```suggestion
To set the scene, add the user’s name at the top of the interview snapshot. For B2B companies, I also recommend adding the job role and company name for context. It’s also handy to add the date of the interview and a high-level summary what the interview was about. If available, add a picture of the user and the company logo, if the context is B2B. This may seem superfluous, but I find it helps trigger my memory when I revisit interviews a few weeks after the fact.
```